src/metabase/pulse/render/body.clj

```diff
@@ -527,14 +527,16 @@
         "line"))
 
 (defn- join-series
-  [names colors types row-seqs y-axis-positions]
-  (vec (for [[card-name card-color card-type rows y-axis-position]
-             (map vector names colors types row-seqs y-axis-positions)]
-         {:name          card-name
-          :color         card-color
```

Comment:
Remove `:color` since all colors will be generated on FE, to match those on app vizs.